manifests/07_deployment.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: CSI_FSGROUP_POLICY
+          value: None
```

Comment:
How is an user supposed to set this env. var in a supported cluster? This operator Deployment will be created by cluster-storage-operator (CSO). If user changes env. var in the Deployment, CSO will overwrite it. They can stop CSO or set it `Unmanaged`, but then I would not call it a supported cluster (e.g. such a cluster cannot be upgraded).

Reply:
Not sure how CSO works (just learning it right now), but I probably see your point: the human operator is not supposed to stop the `manila-csi-operator`, change the value and restart it again with a different env var value, while `cluster-storage-operator` is supposed to manage the lifecycle of the `manila-csi-operator`.
This means that this PR isn't enough to allow people to change the  `CSI_FSGROUP_POLICY` value, because it should be reflected in [1], that drives the behavior of this operator.
Now, the main goal of this patch is not changing the supported policy and allow people to move to a not supported configuration, and the default value (`None`) is enforced is nothing is specified; however, it would be great having this parameter exposed (it was immutable in the `CSIDriver` spec) and have the ability to change it under specific conditions.
So, let's say this patch is not "harmful" and introduces this logic of reading and assigning the `fsGroupPolicy` from an env var: adding the var in [1] is the right (next) move, or am I reading it wrong? 

[1] https://github.com/openshift/cluster-storage-operator/blob/master/assets/csidriveroperators/manila/07_deployment.yaml

Reply:
[1] is indeed the canonical source of Manila operator Deployment. It is instantiated by CSO and here we have the same problem - how users tell CSO to add the env. var there?

If you want to have a supported API, then extension of ClusterCSIDriver would be probably appropriate.
If you want to have an unsupported hack for experiments, stopping CSO and users editing Manila CSI driver deployment manually could be OK.